CONTRIBUTING.md

```diff
@@ -1,5 +1,30 @@
 # Contributing
 
+## Cloning the Repo
+
+This repo contains the submodule `mauth-protocol-test-suite` so requires a flag when initially cloning in order to clone and init submodules.
+
+With Git >= 2.13
+
+```
+git clone --recurse-submodules git@github.com:mdsol/mauth-client-ruby.git
+```
+
+With Git < 2.13
```

Comment:
2.13 has been released in May 2017 and has been out of support for a while, I don't think we need to specify instructions for < 2.13, unless I'm missing something?
https://en.wikipedia.org/wiki/Git#History